app/views/visns/show.html.erb

```diff
@@ -13,7 +13,7 @@
 <div id="visns-show" class="grid-container" data-reload="true">
   <%= render partial: "shared/messages", locals: { small_text: false } %>
   <section id="visn-introduction" class="margin-bottom-3 margin-top-8">
-    <h1 class="font-sans-2xl line-height-46px margin-top-0">VISN <%= @visn.number %>: <%= @visn.name %></h1>
+    <h1 class="font-sans-2xl line-height-46px margin-top-0">VISN<span class="usa-prose-h5 visn-tooltip-header"> <%= render partial: "visns/visns_tooltip", locals: { data_position: 'top'} %></span> <%= @visn.number %>: <%= @visn.name %></h1>
```

Comment:
Can we remove the `usa-prose-h5` class, since the element isn't a header? Also, refer to the comment in the css file, but we can replace `visn-tooltip-header` with `text-top`.